STL_Extension/include/CGAL/Handle_for.h

```diff
@@ -149,12 +149,14 @@ class Handle_for
         return *this;
     }
 
-    ~Handle_for()
+    ~Handle_for() noexcept
```

Comment:
That's useless, it is noexcept by default.

Reply:
It's probably a hang-over from when instead I'd had `CGAL_NOEXCEPT(CGAL_NO_ASSERTIONS_BOOL)` 
I've since found some more uncaught exception issues with the static analysis on the latest runs (where I upgraded from 5.0.3 to master) I will clean this up along with the others and put them in a new PR.